app/decorators/article_decorator.rb

```diff
@@ -23,7 +23,7 @@ def processed_canonical_url
   end
 
   def comments_to_show_count
-    cached_tag_list_array.include?("discuss") ? 75 : 25
+    cached_tag_list_array.include?(I18n.t("decorators.article_decorator.discuss")) ? 75 : 25
```

Comment:
I don't think this tag name will be translated, didn't we punt on this in another PR?

Reply:
I didn't do extensive tracing on how a specific string is used internally. Please point out if I accidentally turn a hardcoded one into i18n in future.